elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -314,7 +318,28 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
                     return new NotFilterExpression(new Predicate(path, Operator.IN, values));
                 }
             } else if (OPERATOR_MAP.containsKey(op)) {
-                return new Predicate(path, OPERATOR_MAP.get(op), values);
+                if (op == isNullOp) {
```

Comment:
We should move this to a separate function I believe to reduce cyclomatic complexity.